src/snmp/Var.cc

```diff
@@ -67,7 +67,7 @@ Snmp::Var::operator += (const Var& var)
         setTimeTicks(asTimeTicks() + var.asTimeTicks());
         break;
     default:
-        debugs(49, DBG_CRITICAL, HERE << "Unsupported type: " << type);
+        debugs(49, DBG_CRITICAL, "Unsupported type: " << type);
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.

I wish we could automatically discover the right prefix, but I do not think this can be done reliably in a simple script. And leaving this visibly broken is better than hiding the problem behind the wrong prefix.